includes/general.admin.inc

```diff
@@ -24,6 +24,13 @@ function islandora_solr_metadata_general_admin_form($form, &$form_state) {
       '#description' => t('Only show unique values from each field. NOTE: Uniqueness is checked on values directly out of Solr. Later formatting of the value could break uniqueness.'),
       '#default_value' => variable_get('islandora_solr_metadata_dedup_values', FALSE),
     ),
+    'islandora_solr_metadata_field_value_separator' => array(
+      '#type' => 'textfield',
+      '#title' => t('Field value separator'),
+      '#description' => t('Characters to separate values in multivalued fields.'),
+      '#default_value' => variable_get('islandora_solr_metadata_field_value_separator', "\n"),
```

Comment:
@qadan which is what this commit was doing if I understand what you're saying

https://github.com/Islandora/islandora_solr_metadata/pull/51/commits/fb2a31be90e33bc359e9dcd0770fd53ae7ad492c